packages/api/src/util/filterEvents.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017-2019 @polkadot/api authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import {EventRecord, SignedBlock} from '@plugnet/types/interfaces';
+
+import {U8a} from '@plugnet/types';
+
+import l from './logging';
+
+export default function filterEvents(
```

Comment:
I would like the utils you added moved to `submittable/utils/*`, so we can easily remove them when we don't need this hack someday in the future

Reply:
Makes sense..done